openapi.yaml

```diff
@@ -19897,6 +19899,38 @@ components:
             x-stoplight:
               id: 02kz5mierypik
             type: string
+    KarpenterNodePoolConsolidation:
+      title: KarpenterNodePoolConsolidation
+      type: object
+      required:
+        - active
+        - days
+        - start_time
+        - duration
+      properties:
+        active:
+          type: boolean
+          default: false
+          nullable: false
+        days:
+          description: | 
+            The days where the consolidation will occur. 
+            The days are indicated in the same manner as for a cron (sunday=0, monday=1, tuesday=2, wednesday=3, thursday=4, friday=5, saturday=6).
+          type: array
+          items:
+            type: number
+            example: 0
+          nullable: false
+        start_time:
+          description: The start date where the consolidation will occur
+          type: string
+          example: "01:30 am"
```

Comment:
translation will be done frontend side: if input is 02:30pm then core receives 14:30

Reply:
TBC @RemiBonnet

Reply:
```suggestion
          example: "22:30"
```